### PR TITLE
added DAVE  - Design Analyse Visualize Export (or Difficult Analysis Very Easy)

### PR DESCRIPTION
**Pull request by RubendeBruin**
@conda-forge/help-python

This is the first contribution to conda-forge. Hope you will accept my contribution.

Tests:
The package is a book-keeping package to make the flow of information between various engineering packages and visualization packages easier for users. This means that there are many optional additional packages. I have tests for all of those locally but can not include them here (software like eg blender is needed to run them) so I have included only the minimal test.

Checklist:
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [X] Source is from official source
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [X] If static libraries are linked in, the license of the static library is packaged.
- [X] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by RubendeBruin** [comment]
@conda-forge/help-python
Linux and OSX builds work, but the win64 build seems to fail because libtiff 4.1.0 can not be obtained from https://conda.anaconda.org/conda-forge/win-64/libtiff-4.1.0-h21b02b4_4.tar.bz2

I do not need this dependancy to build or test, it comes with vtk. Is there a way to exclude this to temporary work around this issue?

afbeelding

**Comment by RubendeBruin** [comment]
@conda-forge/help-python this is ready for review. For the time being I have removed VTK 8.2.0 from the requirements as work-around for the failing libtiff package.

**Comment by RubendeBruin** [comment]
Hi @conda-forge/help-python , the package is now ready for review.

**Comment by npavlovikj** [comment]
@RubendeBruin , when you create recipes for packages already available on "pypi", the preferred way to do that is with `conda-skeleton` (`conda-skeleton pypi usedave`), https://docs.conda.io/projects/conda-build/en/latest/user-guide/tutorials/build-pkgs-skeleton.html. This way, the skeleton of the recipe is there, but you will still need to do manual clean-up and edits. 
Can you please try that? I have few minor comments, that should be fixed when the skeleton is used.

**Comment by RubendeBruin** [comment]
@npavlovikj , I used conda skeleton and made the following changes to the produces .yaml afterwards:

- changed package name to "DAVE" which is the name of the package. The name on pip was already taken.
- Changed license family from "OTHER" to "MOZILLA"
- Requirements, tests

However now the building fails on getting the .tar.gz files from pypi: 
https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=109879&view=logs&j=240f1fee-52bc-5498-a14a-8361bde76ba0&t=78a0099b-3f13-5ae9-c1f0-24f71d1eac20&l=96

what should I do?

File: recipes/DAVE/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: e3ca64989027fa7ebcb7a346864f2e84a5fcf90106aec897b774be1111d6b32a
+ 
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
```
Comment (xhochy) [inline recipes/DAVE/meta.yaml]: The whole build section can be omitted, this is only needed when you need compilers.

File: recipes/DAVE/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  run:
+    - python >=3.5
+    - setuptools
+    - pyscaffold
+    - vtk >=8.2.0
+    - ipython
+    - pyside2
+    - numpy
+    - scipy
+    - matplotlib
```
Comment (scopatz) [inline recipes/DAVE/meta.yaml]: Can this be `matplotlib-base?`

File: recipes/DAVE/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+requirements:
+  host:
+    - python >=3.5
+    - pip
+    - setuptools
+    - pyscaffold
+  run:
+    - python >=3.5
+    - setuptools
```
Comment (scopatz) [inline recipes/DAVE/meta.yaml]: Is setuptools really needed at runtime?

File: recipes/DAVE/meta.yaml (hunk 2 above)
Comment (RubendeBruin) [reply recipes/DAVE/meta.yaml]: Checked and matplotlib-base works as well. Did not know it existed.... Thanks!

File: recipes/DAVE/meta.yaml (hunk 3 above)
Comment (RubendeBruin) [reply recipes/DAVE/meta.yaml]: Nope, you are right.  Removed it removed pyscaffold as well from run.

**Comment by RubendeBruin** [comment]
@scopatz thanks for the comments. implemented, and ready to go

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**